_data/sites.json

```diff
@@ -6549,9 +6549,9 @@
 
     {
         "name": "Fiverr",
-        "url": "https://www.fiverr.com/support_tickets/new/account_and_security",
-        "difficulty": "hard",
-        "notes": "For each checkbox, choose: <ul><li><strong>Category:</strong> \"Data and privacy\"</li><li><strong>Topic:</strong> \"Permanently delete my personal data\"</li><br>Click on the little link \"here\" under the light blue box. Fill the fields and submit request, soon, an account manager will answer you and process your request.",
+        "url": "https://www.fiverr.com/support_tickets/account-and-security/data-and-privacy/remove-personal-data",
+        "difficulty": "easy",
+        "notes": "Go to https://www.fiverr.com/users/YOUR_USERNAME/edit/account, select any reason and click on <strong>deactivate account.</strong>",
```

Comment:
I'm not sure I want to have a non-functional link on the notes, what's the difference between the URL and the one on the notes, in terms of steps?

Also we convert the notes from markdown, so you could use `**deactivate account**` instead of strong HTML tags.

Reply:
The link only works if you log in. So you either use the link and log-in or use the https://www.fiverr.com/users/YOUR_USERNAME/edit/account link directly